python/gen-fixtures.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env sh
+#
+# Copy static files of Python repository into fixtures directory
+#
+set -euo pipefail
+readonly image_name='busybox:latest'
+
+# NOTE: $0 corresponds to the the script's name only in some shells and in some
+# contexts.
+show_help() {
+fmt <<EOF
+usage: gen-fixtures.sh <output_dir> <assets_dir>
+
+Create <output_dir>. Copy Python fixture data from <assets_dir> and place the 
+results into <output_dir>.
+
+EOF
+cat <<EOF
+<output_dir>
+    The directory into which generated fixtures are placed.
+EOF
```

Comment:
Please add a brief description of `<assets_dir>` here.

By the way, the `fmt` on my system is smart enough to handle simple definition lists. I may have been overly-cautious when deciding to use both `fmt` and `cat`. This input:

```sh
fmt -u <<EOF
usage: foo bar

Description of command. Asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf.

definition term 1
    Definition data for definition term 1. Asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf asdf.
definition term 2
    Foo bar biz baz.
EOF
```

Gives this output:

```
usage: foo bar

Description of command. Asdf asdf asdf asdf asdf asdf asdf asdf asdf
asdf asdf asdf asdf asdf asdf asdf.

definition term 1
    Definition data for definition term 1. Asdf asdf asdf asdf asdf asdf
    asdf asdf asdf asdf asdf.
definition term 2
    Foo bar biz baz.
```